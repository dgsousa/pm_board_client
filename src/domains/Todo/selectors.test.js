import Immutable from 'immutable';
import todoSelector from './selectors';

const state = Immutable.fromJS({
  todo: {
    todos: {
      one: 'go to store',
      two: 'setup jest',
    },
  },
});

test('todoSelector', () => {
  const todos = todoSelector(state);
  expect(todos).toMatchSnapshot();
});
