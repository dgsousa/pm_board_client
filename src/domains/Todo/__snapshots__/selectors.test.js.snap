// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todoSelector 1`] = `
Array [
  Object {
    "key": "one",
    "val": "go to store",
  },
  Object {
    "key": "two",
    "val": "setup jest",
  },
]
`;
